### ISSUE TITLE
Forgot how to create Windows Server

### ISSUE THREAD

**Issue by ShaggyG3** [OpenRA/OpenRA#7727]
About 5-8 months ago, I had two-three games goin for about a week for friends. Forgot how to do it, I remember I was referring to a Wiki that actually had some information about Windows but all was replaced with only Linux?

I recall having to take a Shortcut adding parameters in the target field, that was the configuration, later turned it into a service, another theory is that commands we're changed?

https://github.com/OpenRA/OpenRA/wiki/Dedicated

Thanks! Will contribute to documentation when I setup server :)

-----
<b>
I know once I am able to figure out the issue with the crash (exception.log & graphics.log commented below). The server will work.
</b>

**Comment by vmaule212** [comment]
I'm also get this:
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.34209
Exception of type `System.InvalidOperationException`: No suitable renderers were found. Check graphics.log for details.
   at OpenRA.Game.Initialize(Arguments args)
   at OpenRA.Program.Run(String[] args)
   at OpenRA.Program.Main(String[] args)

**Comment by vmaule212** [comment]
Figured it out, please check link in WIki for setting up a OpenRA server in WIndows.

**Comment by ShaggyG3** [comment]
http://www.filedropper.com/servers2

Here are all my scripts, tried to start serves with task manager but no go, what is wrong?

**Comment by Basti504** [comment]
A while back I ran some OpenRA servers on a Windows server, the OpenRa instances were configured as windows services. So I could log off the Windows users and the servers would just continue. 

The actual command line command was:
`Game.Mod=ra Server.Dedicated=true Server.DedicatedLoop=true Server.Name="OpenRA Dedicated 1" Server.ListenPort=1235 Server.ExternalPort=1235 Server.AdvertiseOnline=true Server.Map=ba403f6bcb4cae934335b78be42f714992b3a71a`

Using the NSSM - the Non-Sucking Service Manager (https://nssm.cc/) I created the windows services which ran the OpenRA server thingy. If for some reason the service stopped, it was automatically restarted. The setup worked quite good actually.

Here a screenshot about how the configuration roughly was. I hope this is usefull for your issue.
![openra_as_win_service](https://cloud.githubusercontent.com/assets/5729926/6905261/228cee46-d727-11e4-97f2-4f4da771d24d.png)
